### PR TITLE
get task id from time.php instead of tasks.php

### LINKED ISSUE #7: A small nodejs scraping task
We had an issue with this project that was fixed by a codemill developer, but apparently the task was not properly defined by us, hence we need to solve the problem in another way.

Please read #3 for detailed explanation.

The solution the developer found was to use Anuko's `/tasks.php` in order to scrape tasks data. However, apparently in our production setup, the page `/tasks.php` is available only for admins, so most users can't use Anuko via this proxy.

I noticed that tasks data is also rendered as JS code into the `/time.php` page. The task is therefore exactly as it defined in #3, only this time there's another requirement which is to modify the solution to use scraping of the `/time.php` page.

codemill$20

### PR DESCRIPTION
**Pull request by blitmap**
codemill-id-56f6ba037e64df030099d215 fixes #7

Okay, so I've verified this works on my end.  I'm none too happy that the original implementation and this essentially scrape text from HTML without interpreting the HTML first.  This works, but it's not any cleaner [or worse] than the original.

I had been exploring a better alternative where you create the actual DOM with the jsdom and jquery modules.  Then you would be able to "query" for the task id by traversing to the 3rd `<script>` of the page.

In short, traversing the source of a web page with a `.indexOf()` or a RegExp is just bad.

(I can work on this more if you like)

### CODE REVIEW

**Comment by shaharsol** [comment]
Thanks @blitmap, I don't really care right now about the style of the code, though I hear what you're saying. I'm going to test it soon, but just from a quick look, I have a question: what if two different projects have tasks named the same? Will it pick the correct one? i.e in correlation of the project name that was passed by the user?

**Comment by shaharsol** [comment]
mmm... I think you can ignore my question. It seems they'd be getting the same ID...

**Comment by blitmap** [comment]
You are awfully insightful (asking the right questions) for someone who tasks others with coding.  I have to wonder why a smart guy like you isn't just writing this himself. :3

Let me know if you need anything changed - was pleasure working with you again!  :smile:
